packages/ui/src/components/editor/properties/model/index.tsx

```diff
@@ -108,7 +108,7 @@ export const ModelNodeEditor: EditorComponentType = (props) => {
       return
     }
     exporting.set(true)
-    const fileName = `${srcPath.value}.${exportType.value}`
+    const fileName = `assets/${srcPath.value}.${exportType.value}`
```

Comment:
We don't want to hard-code exports into the "assets" folder, to preserve flexibility with folder structure